test/OrchardCore.Tests/Serializers/JsonSerializerTests.cs

```diff
@@ -28,6 +29,42 @@ public void Deserialize_WhenCalled_ReturnValidUserLoginInfo()
         Assert.Equal("default", obj.ProviderDisplayName);
     }
 
+    [Fact]
```

Comment:
You can use `Theory` with `InlineData` instead of separated facts

Reply:
good idea. Done